services/things/persistence/src/main/java/org/eclipse/ditto/services/things/persistence/actors/strategies/events/AbstractThingEventStrategy.java

```diff
@@ -48,20 +47,14 @@ public Thing handle(final T event, @Nullable final Thing thing, final long revis
         if (null != thing) {
             ThingBuilder.FromCopy thingBuilder = thing.toBuilder()
                     .setRevision(revision)
-                    .setModified(event.getTimestamp().orElse(null));
-            thingBuilder = setMetadata(event, thing, thingBuilder);
+                    .setModified(event.getTimestamp().orElse(null))
+                    .setMetadata(event.getMetadata().orElse(null));
```

Comment:
Isn't the event metadata relativised to the resource path? Is it correct to replace the thing metadata by the event metadata?

Reply:
Nevermind, fixed in subsequent commit
https://github.com/eclipse/ditto/pull/825/commits/f58e8ee5454f7e9d80ee447f2bf968deb9852776